python/src/aiconfig/editor/client/src/components/prompt/PromptContainer.tsx

```diff
@@ -107,7 +107,7 @@ export default memo(function PromptContainer({
   const { classes } = useStyles();
 
   return (
-    <Flex justify="space-between" mt="md">
```

Comment:
Very quickly, what does the `mt` field mean in typescript components?

Reply:
It's margin-top in supported mantine components